app/scripts/views/components/barChart/barChartTemplate.js

```diff
@@ -186,7 +188,15 @@
         width: this.settings.width,
         height: this.settings.height
       });
-
+      groupNdxData = iViz.getGroupNdx(this.opts.groupid);
+      
+      if(this.opts.attrId === 'mutation_count') {
```

Comment:
mutation count data should be added right after the data is ready. It should follow similar `addChart` follow in vueCore.js